common/src/linked_list.c

```diff
@@ -0,0 +1,72 @@
+#include "linked_list.h"
```

Comment:
Do we need get by index and get size of list functions?

Reply:
Both could be useful

Reply:
I'll implement them 👍